assets/webpack.config.js

```diff
@@ -8,10 +8,15 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 
-module.exports = (_env, _options) => ({
+const devtoolOption = (mode) => {
+  if (mode === 'production') return 'source-map';
+  return 'eval';
+};
+
+module.exports = (_, {mode}) => ({
```

Comment:
_Imho_, it would make even more sense to bridge the gap with the Phoenix generator to simplify future enhancement…

https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_assets/webpack.config.js#L10

```js
  const devMode = options.mode !== 'production';
```

Reply:
You’re right, I think I’ll move to this approach.